point_of_sale/belgian_fdm/setup.rst

```diff
@@ -0,0 +1,82 @@
+===================================================
+Setting up the Fiscal Data Module with the Odoo POS
+===================================================
+
+Introduction
+============
+
+The Belgian government requires certain businesses to use a
+government-certified device called a Fiscal Data Module (also known as
+a blackbox). This device works together with the POS application and
+logs certain transactions. On top of that, the used POS application
+must also be certified by the government and must adhere to strict
+standards specified by them. `Odoo 9 is a certified application
+<http://www.systemedecaisseenregistreuse.be/systemes-certifies>`_. More
+information concerning the Fiscal Data Module can be found on `the
+official website <http://www.systemedecaisseenregistreuse.be/>`_.
+
+**Currently our government certificate is only valid for Odoo
+instances on Odoo Online.**
+
+Required hardware
+=================
+
+* A government certified `Fiscal Data Module
+  <http://www.systemedecaisseenregistreuse.be/systemes-certifies#FDM%20certifiés>`_
+  per POS, all of them should work, but the Cleancash SC-B is
+  recommended, you will also need:
+
+  * Serial null modem cable per FDM (`example
+    <http://www.startech.com/Cables/Serial-Parallel-PS-2/DB9-DB25/10-ft-Cross-Wired-Serial-Null-Modem-Cable-DB9-FM~SCNM9FM>`_)
+  * Serial-to-USB adapter per FDM (`example
+    <http://trendnet.com/products/proddetail.asp?prod=265_TU-S9>`_)
+
+* A POSBox per POS
+
+Setup
+=====
+
+POSBox
+------
+
+In order to setup the blackbox, you will first have to ensure that the
+POSBoxes you're using have a recent image on them (support for the FDM
+was added to the POSBox image on the 10th of December 2015). See
+:doc:`../overview/setup` on how to upgrade the POSBox. You can verify
+that the Fiscal Data Module is recognized by the POSBox by going to
+the *Hardware status page* via the POSBox homepage.
+
+.. image:: media/posbox_fdm_hardware_status.png
+    :align: center
+
+Odoo
+----
+
+An Odoo POS is turned into a certified POS by installing the Belgian
+Registered Cash Register module (technical name:
+``pos_blackbox_be``). This installation cannot be undone. After this,
```

Comment:
Can you elaborate why can not be undone? What does it implies if I try to uninstall the app?